gravitee-apim-console-webui/src/components/gio-metadata/gio-metadata.component.ts

```diff
@@ -69,11 +69,11 @@ export class GioMetadataComponent implements OnInit, OnDestroy {
   ngOnInit(): void {
     this.referenceType = this.metadataSaveServices.type;
     this.permissionPrefix = this.referenceType === 'Global' ? 'environment' : this.referenceType.toLowerCase();
-    this.displayedColumns = ['global', 'key', 'name', 'format', 'value', 'defaultValue', 'actions'];
-
+    this.displayedColumns = ['global', 'key', 'name', 'format', 'value', 'actions'];
     this.metadataSaveServices
       .list()
       .pipe(
+        tap((metadata) => metadata['defaultValue'] && this.displayedColumns.push('defaultValue')),
```

Comment:
I don't really understand how this line can work. 🤔  
metadata here is a list right ? . so do metadata['defaultValue'] always return false. 

I'd expect more of a condition on the type like if API add defaultValue column

Reply:
Fixed, I'm using `referenceType` now to define if we want to have `defaultValue`